README.md

```diff
@@ -264,62 +201,80 @@ steps in the next section.
+contributing to searchfox.  Please reach out in #searchfox on
+https://chat.mozilla.org/
+
+- First, follow the [Searchfox AWS docs](docs/aws.md) to ensure you have your
+  credentials working in general and that you can run the
+  `infrastructure/aws/ssh.py` command and successfully get a list of active VMs.
+  In particular, you will probably need to type:
+  - `. env/bin/activate`
+  - `eval $(maws -o awscli --profile default)`
+- Push your changes to your mozsearch branch and your mozsearch-mozilla branch,
+  if appropriate.
+  - It's usually a good idea to explicitly make sure you've saved all your
+    buffers, that `git status` shows no uncommited changes, that
+    `make build-test-repo` runs successfully, and that
+    `git push -f REMOTE BRANCH` says all the commits are already there.
+- Pick what "channel" you are going to use.  Generally, the right answer is the
+  "dev" channel, which will display its results at https://dev.searchfox.org/
+  but it's possible to use and create other channels.  You can see if anyone
+  already has a server up on the "dev" channel by running the
+  `infrastructure/aws/ssh.py` script.
+- Pick what config file you are going to use.  Normally this is "config1.json"
+  which includes the mozilla-central repo and a few other repositories that
+  don't have all the bells and whistles turned on.  You can use a different
+  config file or edit config1.json to not contain repositories you aren't
+  interested in to make things go faster.
+  - You don't have to have your own branch of mozsearch-mozilla!  You can use
+    https://github.com/mozsearch/mozsearch-mozilla and it's fine that it doesn't
+    have a branch with the name of your development branch.  The scripts will
+    automatically fall back to the default branch.
+- If you didn't run a custom m-c try job, you can edit the following:
 ```
-TRYPUSH_REV=<40-char-rev-hash> make trypush
+infrastructure/aws/trigger_indexer.py \
+  https://github.com/some-user/mozsearch \
+  https://github.com/some-user/mozsearch-mozilla \
+  config1.json \
+  some-development-branch \
+  dev
 ```
-This will clone the Mozilla configuration into ~/mozilla-config, and
-generate a reduced config that has just the mozilla-central tree, but
-use the code and artifacts from your try push when building the index.
-It will build the index into a `~/trypush-index` folder to keep it separate
-from any `~/mozilla-index` folders you might have lying around.
-It's very similar to the operations described in the next section
-which will build an index using the latest mozilla-central version with
-searchfox artifacts.
-
-### Testing basic changes
-
-Note: You can also just do `make build-mozilla-repo` in `/vagrant` to have it
-idempotently do the following for you.
-
+- If you did run a custom m-c try job, the only difference is the addition of a
+  `--setenv TRYPUSH_REV=therevision` to to the command.  So this looks like:
```

Comment:
Minor nit: it would be good to specify `therevision` is the full 40-char hash. I think using the abbreviated hash won't work because taskcluster doesn't allow indexing artifacts by abbreviated hash.

Reply:
Good point, thank you!  Addressed in https://github.com/mozsearch/mozsearch/pull/450 hopefully!